lib/lz4.c

```diff
@@ -359,8 +359,11 @@ LZ4_wildCopy32(void* dstPtr, const void* srcPtr, void* dstEnd)
     BYTE* d = (BYTE*)dstPtr;
     const BYTE* s = (const BYTE*)srcPtr;
     BYTE* const e = (BYTE*)dstEnd;
-
+#if defined(__aarch64__) && defined(__GNUC__)
+    do { memcpy(d,s,32); d+=32; s+=32; } while (d<e);
```

Comment:
This change seems incorrect, and can corrupt data.

There's a difference between 2x 16-byte writes, and 1x 32-byte write:
it matters whenever dealing with overlapping matches.

For offset in the [16-32[ range, 2x16-byte will work, and regenerate correct data,
while 1x32-byte will copy some garbage into the last byte(s).

This usage happens at line 195.

To witness the problem, one need a file which happen to generate matches at offset [16-32[,
and an ability to check that decompressed content is the same as original one
(`lz4` internal checksum can do it).

Reply:
Thanks for explationation, I will roll it back.